src/d3d9/d3d9_mem.cpp

```diff
@@ -0,0 +1,352 @@
+#include "d3d9_mem.h"
+#include "../util/util_string.h"
+#include "../util/util_math.h"
+#include "../util/log/log.h"
+#include "../util/util_likely.h"
+#include <utility>
+#include <sysinfoapi.h>
```

Comment:
?

Reply:
Uhhhh, I blame VSCode. :)

Reply:
if `clangd` LSP then something like this may help (see `clangd --help`)
```
{
    "clangd.arguments": [
        "--header-insertion=never",
        "--suggest-missing-includes",
        ...
    ]
}
```

Reply:
I'm on winelib, no problems if you really need sysinfoapi.h on mingw